### PR TITLE
update torch version for cuda9.2 and fix link

### PR DESCRIPTION
**Pull request by bertoni9**
- fix link for other page
- update torch version suggestion
- remove print connections

### CODE REVIEW

File: guide/plugins_apollocar3d.ipynb [hunk 1]
```diff
@@ -210,14 +176,14 @@
     "\n",
     "## Everything else\n",
     "All pifpaf options and commands still stand, read more in the\n",
-    "[OpenPifPaf guide](https://openpifpaf.github.io/openpifpaf/intro.html). If you are interested in training your own dataset, read the section on a [Custom Plugin](https://openpifpaf.github.io/openpifpaf/plugins_custom.html)."
+    "[OpenPifPaf guide](https://openpifpaf.github.io/openpifpaf/intro.html). If you are interested in training your own dataset, read the section on a {doc}`custom dataset <plugins_custom>`."
```
Comment (svenkreiss) [inline guide/plugins_apollocar3d.ipynb:179]: Regarding the first link: This is now a page of the Guide and I find it a bit odd to link to the Guide so explicitly here. In any case, this link is also broken.

**Review by svenkreiss — COMMENTED** [review]
Thanks! Just one minor thing.

File: guide/plugins_apollocar3d.ipynb (hunk 1 above)
Comment (bertoni9) [reply guide/plugins_apollocar3d.ipynb]: Thanks, I agree. I have just removed the link.

**Comment by svenkreiss** [comment]
Merci!

**Pull request merged**